website/Makefile

```diff
@@ -1,6 +1,6 @@
 deps:
-	rsync -av ../* root/ --exclude=website --exclude=root/design/colors
-	rsync -av 'root/design/' 'static/design/' --exclude='#*'
+	rsync -av ../ content/ --include='assessments' --include='governance' --include='governance/related-groups' --include='supply-chain-security' --include='*.md' --exclude='*'
+	rsync -av '../design/' 'static/design/' --exclude='#*'
```

Comment:
LGTM.

I went with the previous approach because the hugo templating and makefile rsync took me a while to harmonize, but I like this change.

Reply:
I thought adding the right folders like you did here would not cause broken links issue anymore but it seems like this one: https://deploy-preview-1245--tag-security.netlify.app/supply-chain-security/readme/ has some broken links. Is it deploy-preview being funky or something else?

Reply:
There are definitely more broken links to be resolved. I focused on ensuring we resolved the links from the homepage first and foremost (closing the original issue).

Hereafter I believe there is more to do with regards to strategy on syncing the repo directories into the website directory and ensuring links are resolving appropriately.

Reply:
I agree with this. It might be worth us setting aside some time next week to just hammer this out together.

Reply:
Sounds good. Will merge this as is for now. You both can discuss more and come up with a strategy and we can then share it with the group :) for more feedback!